hc/front/tests/test_add_check.py

```diff
@@ -11,4 +11,14 @@ def test_it_works(self):
         self.assertRedirects(r, "/checks/")
         assert Check.objects.count() == 1
 
-    ### Test that team access works
+    def test_team_access_works(self):
+        url = "/checks/add/"
+        self.client.login(username="alice@example.org", password="password")
+        self.client.post(url)
+        check = Check.objects.filter()
```

Comment:
why use .filter() when there's nothing specific you're filtering? What of .all()?